pupynere.py

```diff
@@ -346,19 +346,25 @@ def createVariable(self, name, type, dimensions=None, attributes=None):
 
         shape = tuple([self.dimensions[dim] for dim in dimensions])
         shape_ = tuple([dim or 0 for dim in shape])  # replace None with 0 for numpy
+        isrec = None in shape
```

Comment:
Does "isrec" stand for "is record variable"? It would help to make the identifier more explicit and/or document the name. (I prefer the more explicit identifier.)

Reply:
Yes. The name "isrec" is used elsewhere in the file, so I used it here for consistency.